tests/finder-frontend.spec.js

```diff
@@ -1,7 +1,10 @@
 import { expect } from "@playwright/test";
 import { test } from "../lib/cachebust-test";
+import { trackBrowserErrors } from "../lib/track-browser-errors";
 
 test.describe("Finder frontend", { tag: ["@app-finder-frontend"] }, () => {
+  trackBrowserErrors(test);
```

Comment:
Do we know of a way we can apply this to everything? I guess the equivalent of a beforeEach for everything in the suite?

There is setup files - an example is the auth.setup.js  -perhaps those type of files offer a hook to do a global beforeEach.